lab

```diff
@@ -56,6 +61,23 @@ else
                 -e create_env_file=$WORKING_DIR/create_env.sh"
 fi
 
+function configure_ssh_jump_host() {
```

Comment:
This is a copy&paste from osp-workshop-2024.sh - maybe we should move it to a common file and source it in the `lab` to avoid code duplication? Or is there a reason `lab` should not source external files?

Reply:
I didn't want to source external files from lab script because I though it is better to keep that "lab" script to be independent so that potentially it can be simply moved to /usr/local/bin and used from there by students without anything else

Reply:
fair enough